"Error: Cannot find module 'chalk'" after `npm ci --only=production`

**Describe the bug**
When using node-ocpua after `npm ci --only=production` we get an error `Error: Cannot find module 'chalk'`

**Step by step instruction to reproduce**

Steps to reproduce the behavior:

1. npm i -s node-opcua
2. npm ci --only=production
3. run any opcua code

**Context**

Chalk should be added as dependency not devDependency in node-opcua-debug

- **Node:**
  `node --version` = 16.x

--- a/packages/node-opcua-debug/package.json
+++ b/packages/node-opcua-debug/package.json
@@ -8,12 +8,12 @@
         "test": "mocha"
     },
     "dependencies": {
+        "chalk": "4.1.2",
         "hexy": "0.3.4",
         "node-opcua-assert": "2.66.0",
         "node-opcua-buffer-utils": "2.69.0"
     },
     "devDependencies": {
-        "chalk": "4.1.2",
         "should": "^13.2.3"
     },
     "main": "./dist/index.js",

